bears/typescript/TSLintBear.py

```diff
@@ -26,16 +52,170 @@ class TSLintBear:
+            ``@Directive`` and ``@Component`` decorators.
+        :param use_output_property_decorators:
+            Use ``@output`` instead of the output properties of the
+            ``@Directive`` and ``@Component`` decorators.
+        :param use_host_property_decorators:
+            Use ``@HostListener`` and ``@HostBinding`` instead of the host
+            property of the ``@Directive`` and ``@Component`` decorators.
+        :param allow_attribute_parameter_decorators:
+            Allows using ``attribute`` decorator.
+        :param allow_input_rename:
+            Allows renaming directives input properties.
+        :param allow_output_rename:
+            Allows renaming directives output properties.
+        :param allow_forward_references:
+            Allows the use of ``forwardRef``.
+            Example:
+            ::
+                @Component({
+                  directives: [forwardRef(()=>NameService)]
+                })
+                class Test {}
+                class NameService {}
+        :param use_life_cycle_interface:
+            Use life cycle hooks with their corresponding interfaces.
+        :param pipe_naming_convention:
+            Naming convention for pipes (``hyphen``, ``camel`` (default),
+            ``snake``).
+        :param pipe_prefix:
+            Use a custom prefix for our pipe.
+        :param force_pipe_prefix:
+            Enforces the use of a custom prefix for pipes.
+        :param component_class_suffix:
+            Use ``Component`` as component classes name's suffix when
+            the ``@Component`` decorator is used, ``Pipe`` when the ``@Pipe``
+            decorator is used, and ``Empty`` when the class is empty.
+        :param directive_class_suffix:
+            Use ``Directive`` as directive classes name's suffix when
+            the ``@Directive`` decorator is used, ``Pipe`` when the ``@Pipe``
+            decorator is used, and ``Empty`` when the class is empty.
+            is used and ``Empty`` when the class is empty.
+        :param check_import_destructuring_spacing:
+            Checks valid spacing inside ``import`` statements.
+
         """
         args = ('--format', 'json')
         if tslint_config:
             args += ('--config', tslint_config)
         if rules_dir:
             args += ('--rules-dir', rules_dir)
+        if not tslint_config:
+            _tslint_directive_selector_naming_convention = (
+                map_naming_convention_to_tslint(
+                   directive_selectors_naming_convention))
+            _tslint_component_selector_naming_convention = (
+                map_naming_convention_to_tslint(
+                   component_selectors_naming_convention))
+            rules = {
+                     "directive-selector-name": [
+                         (False
+                          if _tslint_directive_selector_naming_convention is ''
+                          else True),
```

Comment:
just `_tslint_directive_selector_naming_convention is not ''` without the inline if is sufficient, also for the other conditions